eng/common/tools.ps1

```diff
@@ -285,6 +285,22 @@ function InitializeXCopyMSBuild([string]$packageVersion, [bool]$install) {
   return Join-Path $packageDir "tools"
 }
 
+function EnablePreviewSdks() {
```

Comment:
@brettfo I figured out what was happening. Once it found VS on the CI machine, it didn't have preview sdks enabled, which is why it couldn't find netcoreapp 3.0. This code here is from Roslyn's build, minus the msbuild check.